src/nrtk_explorer/app/transforms.py

```diff
@@ -389,7 +391,7 @@ def apply_ui(self):
             self._parameters_app.transform_apply_ui()
 
     def dataset_widget(self):
-        ImageList(self.on_scroll, self.on_hover)
+        ImageList(self.server, self.on_scroll, self.on_hover)
```

Comment:
Just need the methods not the server

Reply:
unless it is a child server and that is important. In which case, you can pass it as `trame_server=self.server` and let the super class handle it.